osu.Game.Rulesets.Osu/Skinning/LegacyMainCirclePiece.cs

```diff
@@ -44,6 +46,25 @@ private void load(DrawableHitObject drawableObject, ISkinSource skin)
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                 },
+                new KiaiFlash
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Size = new Vector2(OsuLegacySkinTransformer.LEGACY_CIRCLE_RADIUS / 64),
+
+                    Masking = true,
+                    CornerRadius = Size.X / 2,
+                    CornerExponent = 2,
+
+                    Intensity = 0.1f,
+                    Child = new Box
```

Comment:
I'm confused as to why this child box isn't in the `KiaiFlash` class itself. Also not sure a legacy implementation should be made with a separate design unless it matches stable 1:1 (this code doesn't look to be).